library/Cube/Cube.php

```diff
@@ -4,11 +4,18 @@
 
 namespace Icinga\Module\Cube;
 
+use Icinga\Application\Modules\Module;
 use Icinga\Exception\IcingaException;
+use Icinga\Module\Cube\ProvidedHook\Icingadb\IcingadbSupport;
 use Icinga\Web\View;
 
 abstract class Cube
 {
+    public const SLICE_PREFIX = 'slice.';
+
+    /** @var bool Whether the icingadb backend is in use */
+    public const IS_USING_ICINGADB = false;
```

Comment:
Why is this only overridden by `IcingaDbCube` and not also in `IdoCube`? With the monitoring module this *shortcut* isn't used currently.

Reply:
This isn't resolved if you still initialize this with `false` and not change the check below.